### ISSUE TITLE
`random: true` in config json doesn't appear to work

### ISSUE THREAD

**Issue by jasonkeene** [jasmine/jasmine#1120]
I just started using jasmine and am used to having specs randomized in other languages. I'd prefer not to have to type `--random=true` to have this. Here is my config file:

```json
{
  "spec_dir": "spec",
  "spec_files": [
    "**/*[sS]pec.js"
  ],
  "helpers": [
    "helpers/**/*.js"
  ],
  "stopSpecOnExpectationFailure": true,
  "random": true
}
```

Here is my `jasmine version` output:

```
$ jasmine version
jasmine v2.4.1
jasmine-core v2.4.1
```

Here is what `JASMINE_CONFIG_PATH` is set to:

```
$ echo $JASMINE_CONFIG_PATH
spec/config/jasmine.json
```

And here is where my config file is located:

```
$ find spec | grep json
spec/config/jasmine.json
```

I also tried putting the config where `jasmine init` placed it under `spec/support/jasmine.json` and setting the environment variable but no go. I'm running `jasmine` in a loop forever and it doesn't hit a failure I wrote that depends on the ordering of tests. When I run `jasmine --random=true` it shows up 50% of the time as expected. Not sure what I'm missing here.

**Comment by zack** [comment]
The documentation at https://jasmine.github.io/2.5/node.html shows `random: ...`, where you have `"random": ...`.

**Comment by jasonkeene** [comment]
@zack I would say the docs are wrong in that case since without the " it is invalid json. I seriously doubt they'd load this file by requiring it as JavaScript. Even if they did both would result in the same object.

**Comment by slackersoft** [comment]
It looks like this was probably fixed in jasmine/jasmine-npm#65 which is in the latest release (2.5.x). Please update and give it a shot.

Closing this. Thanks for using Jasmine!
